tests/ml/test_ml_model_pytest.py

```diff
@@ -263,10 +263,12 @@ def test_xgb_classifier(self, compress_model_definition, multi_class):
             training_data = datasets.make_classification(
                 n_features=5, n_classes=3, n_informative=3
             )
-            classifier = XGBClassifier(booster="gbtree", objective="multi:softmax")
+            classifier = XGBClassifier(
+                booster="gbtree", objective="multi:softmax", use_label_encoder=False
+            )
         else:
             training_data = datasets.make_classification(n_features=5)
-            classifier = XGBClassifier(booster="gbtree")
+            classifier = XGBClassifier(booster="gbtree", use_label_encoder=False)
```

Comment:
@benwtrent Is this fine for you?

Reply:
Yes, this should be ok. Especially if we get a ✅ build. We don't automatically translate encodings from the models anyways.